src/scenes/Home/components/Clock/index.js

```diff
@@ -51,12 +51,10 @@ class Clock extends Component {
   render() {
 
     return (
-      <div className="center">
         <div className="app-container clock">
           <div className="clock-time">{this.state.date.toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'})}</div>
```

Comment:
This is ugly (this.state.date.toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'})) ... put that in a constant above, and use the constant that will make it easire to read :) also the other state stuff, use destructuring above and use them with clear one named variables in the view so that we can read easier